templates/interfaces.j2

```diff
@@ -160,12 +160,31 @@ post-up ovs-vsctl {{''
 # set default route
 post-up ip r replace default via {{ network_management_default_gateway }}{% if network_management_default_src_address is defined %} src {{ network_management_default_src_address }}{% endif %}
 {% endif %}
+{% if network_management_tables is defined %}
```

Comment:
`network_management_tables` is always defined because of the defaults.
I think using the length filter would be better.
```suggestion
{% if network_management_tables | length > 0 %}
```

Reply:
Has been done.